ReactWindows/ReactNative.Interfaces/Bridge/IReactContext.cs

```diff
@@ -0,0 +1,190 @@
+using System;
+
+namespace ReactNative.Bridge
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// Abstract context wrapper for the React instance to manage
+    /// lifecycle events.
+    /// </summary>
+    public interface IReactContext : IAsyncDisposable
+    {
```

Comment:
We'll want to minimize what we expose to modules. I'm thinking the interface only needs to expose
- Get*Module<T>
- Add/Remove*Listener
- IsOn*Thread
- AssertOn*Thread
- RunOn*Thread
- HandleException

Reply:
#fixed

Reply:
We should do an inventory of native module add-ons that support RN Windows and make sure they can work with this reduced surface area. linear-gradient, video, ..?